src/Web/AzureDeveloperCliCredential.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Azure.Core;
```

Comment:
Instead of this, please reference the 1.9.0-beta.2 version of Azure.Identity and use the AzureDeveloperCliCredential  https://github.com/Azure/azure-sdk-for-net/blob/Azure.Identity_1.9.0-beta.2/sdk/identity/Azure.Identity/CHANGELOG.md